pdns/dbdnsseckeeper.cc

```diff
@@ -144,8 +144,8 @@ bool DNSSECKeeper::clearMetaCache(const DNSName& name)
 
 void DNSSECKeeper::clearCaches(const DNSName& name)
 {
-  clearKeyCache(name);
-  clearMetaCache(name);
+  if (!clearKeyCache(name) || !clearMetaCache(name))
+     g_log<<Logger::Warning<<"Failed to clear cache for "<<name<<std::endl;
```

Comment:
This is dead code. clearKeyCache and clearMetaCache always return true.

Reply:
wonder why they bother returning value at all? is it some future-planning thing?

Reply:
as discussed on IRC: the `return addKey(name, dspk, id, active, published) && clearKeyCache(name);` construction that is used in 15 variations throughout dbdnsseckeeper.cc needs the `true` return value.